Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -954,11 +954,12 @@ private void DestroyAllServices()
             // Unregister core services (active systems)
             // We need to destroy services in backwards order as those which are initialized 
             // later may rely on those which are initialized first.
-            var orderedActiveSystems = activeSystems.OrderByDescending(m => m.Value.Priority);
+            var orderedActiveSystems = activeSystems.OrderByDescending(m => m.Value.Priority).ToArray();
```

Comment:
Does this new `ToArray()` call not create different garbage or have a perf impact?

Reply:
Yea I know, we could possibly leave this as a foreach loop. 

I made this a ToArray() because using ElementAt() extension supposedly can run in O(N). I'll switch back as it's more likely we'll GC more as our services list grows vs the fixed 40B